sirepo/package_data/static/js/raydata.js

```diff
@@ -318,6 +319,7 @@ SIREPO.app.directive('scansTable', function() {
                   </thead>
                   <tbody>
                     <tr ng-repeat="s in scans | orderBy:orderByColumn:reverseSortScans" data-ng-click="setSelectedScan(s)">
+                      <td><button class="btn btn-info btn-xs" data-ng-click="showRunLogModal(s, $event)">run log</button></td>
```

Comment:
How about "View Log" instead of "run log"? I think its a bit clearer what will happen (viewing something instead of running something) and we tend to capitalize words like that elsewhere in Sirepo (ex "Save Changes")

Reply:
4d8ecf81fb4e98997af2c3b6de70315095700cc9